### DISCUSSION TITLE
Welcome to OpenSplat Discussions!

### DISCUSSION THREAD

**Discussion by pierotofy** [pierotofy/OpenSplat#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wbg-png** [comment]
Trying OpenSplat for the first time (paid version), looking for an alternative to Postshot.

I've tried creating the Colmap data using RealityScan and Colmap itself, but am getting an Invalid project folder error.  I've used RealityScan a bit, but am completely new to Colmap.

How should the Colmap data directory be arranged?  Everything in one folder, separate image & sparce folders, ...?
